frontend/src/App.test.tsx

```diff
@@ -407,30 +409,6 @@ describe("App", () => {
       props.hostCommunication.currentState.requestedPageScriptHash
     ).toBeNull()
   })
-
-  it("should return the auth token set in hostCommunication from getHostAuthToken", () => {
-    const props = getProps()
-    const wrapper = shallow(<App {...props} />)
-
-    // Use setProps (vs setting the auth token on component rendered) to
-    // simulate the auth token changing after the withHostCommunication hoc
-    // initially loads.
-    wrapper.setProps(
-      getProps({
-        hostCommunication: getHostCommunicationProp({
-          currentState: getHostCommunicationState({
-            authToken: "verySecureAuthToken",
-          }),
-        }),
-      })
-    )
-    wrapper.update()
-
-    const instance = wrapper.instance() as App
-
-    // @ts-ignore
-    expect(instance.getHostAuthToken()).toBe("verySecureAuthToken")
-  })
```

Comment:
This test + the corresponding App.tsx function were removed because we might as well just pass the `withHostCommunication` function directly instead of having a thin wrapper around it